content/en/digital_certificate.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely. 
+Certificates authenticate the user through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservice](https://glossary.cncf.io/microservices/) app. 
```

Comment:
Looks like you might have missed this comment in my previews feedback:

This feels like a bit too much information: "through encryption methods and a set of roles, policies, and resources used to manage, use and revoke certificates and encryptions." It will probably raise more questions than answering them. Should we skip this? Maybe this is enough?

> Certificates authenticate users through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservices](https://glossary.cncf.io/microservices/) app.

Reply:
We need a line between the two paragraphs